src/vorta/borg/compact.py

```diff
@@ -43,11 +43,6 @@ def prepare(cls, profile):
         else:
             ret['ok'] = False  # Set back to false, so we can do our own checks here.
 
-        if not borg_compat.check('COMPACT_SUBCOMMAND'):
-            ret['ok'] = False
-            ret['message'] = trans_late('messages', 'This feature needs Borg 1.2.0 or higher.')
-            return ret
-
```

Comment:
Instead of removing this you could raise an Exception 🤷

Reply:
Right. In case the code is somehow run with the wrong version. But how would that work if the same check already runs elsewhere? Who knows..

Reply:
Hmm, even I thought it might be redundant but ig good to have it.